src/components/VLicenseElements.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <ul>
+    <li
+      v-for="(element, index) in elements"
+      :key="index"
```

Comment:
Are the elements unique? If so, can't we just use the `element` as the key rather than the index?

Reply:
Resolved in 0a25092.